test cases/common/178 bothlibraries/meson.build

```diff
@@ -49,14 +49,27 @@ test('runtest-shared-2', exe_shared2)
 test('runtest-static-2', exe_static2)
 test('runtest-both-2', exe_both2)
 
+copy_py = find_program('copy.py')
```

Comment:
fs.copyfile() does that for you.

Reply:
I can do that, but copy.py just got moved up here. It's also used in `generator()` calls, so we can't replace it in that case.

Reply:
Oh, did not notice it's used for generator too. That's fine then, ok.